image_proc/src/rectify.cpp

```diff
@@ -77,13 +77,29 @@ RectifyNode::RectifyNode(const rclcpp::NodeOptions& options)
       }
       return result;
     };
+
+  // Declare parameters
+  this->declare_parameter("camera_namespace", rclcpp::ParameterValue());
```

Comment:
Can we set these as logical defaults? image_color, image_mono, camera_namespace being camera, etc. Right now if you basically dont use the launch file it'll never work